src/components/SelectionList/ChatListItem.tsx

```diff
@@ -92,13 +88,17 @@ function ChatListItem<TItem extends ListItem>({
                             <ReportActionItemDate created={reportActionItem.created ?? ''} />
                         </View>
                         <View style={styles.chatItemMessage}>
-                            <ReportActionItemFragment
-                                fragment={fragment}
-                                actionName={CONST.REPORT.ACTIONS.TYPE.ADD_COMMENT}
-                                source=""
-                                accountID={from.accountID}
-                                isFragmentContainingDisplayName
-                            />
+                            {reportActionItem.message.map((fragment, index) => (
+                                <ReportActionItemFragment
+                                    // eslint-disable-next-line react/no-array-index-key
+                                    key={`actionFragment-${reportActionItem.reportActionID}-${index}`}
```

Comment:
The reportActionID is unique in our database so there's no need to use the array index